core/app/models/spree/product.rb

```diff
@@ -75,6 +75,20 @@ def find_or_build_master
       master || build_master
     end
 
+    def digital_assets
```

Comment:
Can this be rewritten using AR associations or via Arel? Right now it's uncached so it can't be lazy loaded, and it will run 3 times on the page where I saw it in `engine_storefront_backend_ui`.